test/scripts/fullNetworkSetup.ts

```diff
@@ -0,0 +1,148 @@
+import { MultiAddress } from "@polkadot-api/descriptors";
+import { accounts, getZombieClients, waitForChain, waitForRandomness } from "../util";
+import { Binary } from "polkadot-api";
+import { isEqual } from "lodash";
+
+const idealExecutorParams = [
+  {
+    type: "MaxMemoryPages",
+    value: 8192,
+  },
+  {
+    type: "PvfExecTimeout",
+    value: [
+      {
+        type: "Backing",
+        value: undefined,
+      },
+      2500n,
+    ],
+  },
+  {
+    type: "PvfExecTimeout",
+    value: [
+      {
+        type: "Approval",
+        value: undefined,
+      },
+      15000n,
+    ],
+  },
+];
+
+const { relayClient, relayApi, shClient, storageApi } = await getZombieClients({
+  relayWs: "ws://127.0.0.1:31000",
+  // relayWs: "wss://rococo-rpc.polkadot.io",
+  shWs: "ws://127.0.0.1:32000",
+});
+
+async function main() {
+  await waitForChain(relayClient);
+
+  // Check if executor parameters are set
+
+  const { executor_params } = await relayApi.query.Configuration.ActiveConfig.getValue();
```

Comment:
Here and in every other line that uses `relayApi`, I'm getting an error like this:
```
'storageApi.query.System.Account' is of type 'unknown'.
Property 'System' comes from an index signature, so it must be accessed with ['System'].
```

I presume it may be something related to my TS config, but just flagging it in case.

Reply:
When you merged the latest main, that results in a different binary to execute against - which changes the SCALE file. New scale file implies a new type interface, so everything breaks.

Latest commit should fix this which includes updated scale files 🤞